presto-native-execution/scripts/setup-centos.sh

```diff
@@ -63,26 +63,6 @@ export COMPILER_FLAGS=$(echo -n $(get_cxx_flags $CPU_TARGET))
   ln -s /usr/local/gperf/3_1/bin/gperf /usr/local/bin/
 )
 
-(
```

Comment:
We don't need libsodium since we install the package `libsodium-devel` in the Velox script.

Reply:
Good point will remove.